### PR TITLE
feat: prioritize user languages in language picker

### LINKED ISSUE #1548: Prioritize known languages in language selector
As discussed in #1247, user can know several languages. Now he can opt-out them from translating. 

Maybe it can be improved further by prioritizing them in language selector menu, when composing new message

Screenshot 2023-02-01 at 02 31 17

### PR DESCRIPTION
**Pull request by niklaswolf**
closes #1548 

if the user has configured one or more language not to translate (aka known languages), these languages get prioritized in the language picker because these are most likely the languages the user is searching for.

Bildschirmfoto 2023-02-03 um 23 45 58

### CODE REVIEW

**Review by edimitchel — APPROVED** [review]

File: components/publish/PublishLanguagePicker.vue [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <script setup lang="ts">
 import Fuse from 'fuse.js'
+import { ComputedRef } from 'vue'
```
Comment (userquin) [inline components/publish/PublishLanguagePicker.vue:3]: can this be removed?

Comment (danielroe) [reply components/publish/PublishLanguagePicker.vue:3]:
```suggestion
```

File: components/publish/PublishLanguagePicker.vue [hunk 2]
```diff
@@ -38,6 +52,18 @@ function chooseLanguage(language: string) {
       >
     </div>
     <div max-h-40vh overflow-auto>
+      <template v-if="!languageKeyword.trim()">
```
Comment (danielroe) [inline components/publish/PublishLanguagePicker.vue:55]: Should this be something like?

```suggestion
      <template v-if="preferredLanguages.length">
```

Comment (niklaswolf) [reply components/publish/PublishLanguagePicker.vue:55]: No, my intention was to hide this completely when the user uses the searchbar

Comment (danielroe) [reply components/publish/PublishLanguagePicker.vue:55]: Ahh, understood 👍

**Review by danielroe — APPROVED** [review]

**Pull request merged**